spec/views/gsis/index.html.slim_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+require 'helpers/json_format_helper'
+
+describe 'gsis/index' do
+  let(:model_class) { User }
+  let(:variable_to_assign) { :gsis }
+  let(:expected_fields) { %w(id name email) }
+
+  before(:each) do
+    assign(:course, stub_model(Course, attributes_for(:course)))
+  end
+
+  it_behaves_like 'a JSON index view:'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.